homeassistant/components/hikvision/manifest.json

```diff
@@ -8,5 +8,5 @@
   "iot_class": "local_push",
   "loggers": ["pyhik"],
   "quality_scale": "legacy",
-  "requirements": ["pyHik==0.3.2"]
+  "requirements": ["pyHik==0.3.4"]
```

Comment:
Can we bump in a separate PR? Can you also create a separate PR where you add yourself as codeowner as you are now helping a lot with the integration?